wordfreq_builder/wordfreq_builder/tokenizers.py

```diff
@@ -42,6 +45,26 @@ def cld2_surface_tokenizer(text):
     return lang, tokens
 
 
+# Low-frequency languages tend to be detected incorrectly. Keep a limited
+# list of languages we're allowed to use here.
+KEEP_THESE_LANGUAGES = {
+    'ar', 'de', 'el', 'en', 'es', 'fr', 'hr', 'id', 'it', 'ja', 'ko', 'ms',
+    'nl', 'pl', 'pt', 'ro', 'ru', 'sv'
+}
+
+
+def cld2_reddit_tokenizer(text):
+    text = URL_RE.sub('', text)
+    text = MARKDOWN_URL_RESIDUE_RE.sub(']', text)
+
+    lang = cld2_detect_language(text)
+    if lang not in KEEP_THESE_LANGUAGES:
```

Comment:
Definitely that. Reddit is fundamentally an English website, though it has pockets where people speak Japanese or Swedish or whatever. If cld2 thinks someone's comment is in Crimean Tatar, it's probably just in English with an unexpected word in it.